adserver/templates/adserver/email/flight_wrapup.html

```diff
@@ -3,11 +3,11 @@
 
 
 {% block body %}
-<p>{% blocktrans %}Hi,{% endblocktrans %}</p>
+<p>{% blocktrans %}Hello {{ advertiser.name }} team,{% endblocktrans %}</p>
 
-<p>{% blocktrans with site_name=site.name %}Thank you for advertising with {{ site_name }}!{% endblocktrans %}</p>
+<p>{% blocktrans with site_name=site.name %}Thanks again for advertising with {{ site_name }}!{% endblocktrans %}</p>
 
-<p>{% blocktrans %}Your ad flight has wrapped up:{% endblocktrans %}</p>
+<p>{% blocktrans %}You have {{ completed_flights }} flight{{ completed_flights|pluralize }} that have finished:{% endblocktrans %}</p>
```

Comment:
I don't know if this will work correctly. I don't think you can call a filter inside a `blocktrans`, can you?